web/libs/core/src/blocks/SampleDatasetSelect/SampleDatasetSelect.tsx

```diff
@@ -0,0 +1,48 @@
+import { Select, SelectContent, SelectItem, SelectTrigger } from "@humansignal/shad/components/ui/select";
+import { useCallback, useMemo } from "react";
+
+type Sample = {
+  title: string;
+  url: string;
+  description: string;
+};
+
+export function SampleDatasetSelect({
+  samples,
+  sample,
+  onSampleApplied,
+}: {
+  samples: Sample[];
+  sample?: Sample;
+  onSampleApplied: (sample?: Sample) => void;
+}) {
+  const title = useMemo(() => {
+    return sample?.title ?? "Select sample";
+  }, [sample]);
+
+  const onSelect = useCallback(
+    (value: string) => {
+      onSampleApplied(samples.find((s) => s.url === value));
+    },
+    [samples, onSampleApplied],
+  );
+
+  return (
+    <div className="flex gap-3 items-center">
+      <span className="text-lsNeutralContentSubtler">or use a sample dataset</span>
```

Comment:
Small detail: this text "or use a sample dataset" has a different shade of gray as the "or" right before (between the Add URL and Upload files buttons). I guess they should be the same color unless design tells otherwise right?

Reply:
the color used for the "or" before is `--sand_600`

`colors.scss`
```
  --sand_600: #6B6860;
```

`tailwind.config.js`
```
lsNeutralContentSubtler: "#6B6860",
```a

Reply:
You are totally right, my bad